jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/security/authentication/FormAuthenticator.java

```diff
@@ -235,6 +235,7 @@ public Authentication validateRequest(Request req, Response res, Callback callba
         // Handle a request for authentication.
         if (jSecurityCheck)
         {
+            req.accept();
```

Comment:
Everywhere else you've tested if the request is accepted first before accepting it. So either need to do the same here, or if request.accept ignores subsequent accepts then you can remove all of the tests prior to the call.

Reply:
Because you can't do deferred FORM authentication, there is never a case where the request is already accepted